Package.swift

```diff
@@ -359,7 +359,7 @@ let targets: [Target] = [
 
 let package = Package(
     name: "tuist",
-    platforms: [.macOS(.v12)],
+    platforms: [.macOS(.v13)],
```

Comment:
`Command` package supports only macOS 13 and higher. Given macOS 12 is really old, I think it should be safe to bump it.